test/extended/dr/quorum_restore.go

```diff
@@ -137,17 +138,22 @@ var _ = g.Describe("[sig-etcd][Feature:DisasterRecovery][Disruptive]", func() {
 						// there is a small chance the cluster restores the default replica size during
 						// this loop process, so keep forcing quorum guard to be zero, without failing on
 						// errors
-						scaleEtcdQuorum(pollClient, 0)
+						err = scaleEtcdQuorum(pollClient, 0)
```

Comment:
inline the err into the if unless you're using it outside this loop (which i don't see)

Reply:
Done.